gradlew

```diff
@@ -1,25 +1,43 @@
-#!/usr/bin/env bash
+#!/usr/bin/env sh
```

Comment:
:hammer: this file is 100% from running `./gradlew wrapper --gradle-version 5.1.1 --distribution-type all`.